tmt/base.py

```diff
@@ -725,10 +725,19 @@ def lint(self):
         # Explore all available plugins
         tmt.plugins.explore()
 
+        invalid_keys = self.lint_keys(list(self.steps(True, True, True)))
```

Comment:
Just a nitpick but since `self.steps()` contains named arguments, I'd much rather see here `self.steps(enabled=True, disabled=True, names=True)` since it brings more clarity to what those arguments mean without having to look at the method. This is also consistent with how the method is used everywhere else.

Also, this way you are only checking the steps and the attributes inherited from `Core` class but according to the specification [1], there can be more plan-specific attributes, e.g. `environment`.

[1] https://tmt.readthedocs.io/en/stable/spec/plans.html